scripts/README.md

```diff
@@ -56,7 +56,7 @@ Alternatively, you can call each of the scripts individually, e.g.
 sbatch trainer_scripts/drop-only_0_0_trainer.sh
 ```
 
-to train models for that particular dataset pair, where `drop-only_0_0_trainer` is one of the bash scripts generated for the `drop-only` experiment group, with dataset pair index `0` and seed index `0`. Note that logging is performed with wandb and so you will need this set up on your machine in order to proceed.
+to train models for that particular dataset pair, where `drop-only_0_0_trainer` is one of the bash scripts generated for the `drop-only` experiment group, with dataset pair index `0` and seed index `0`. Note that logging is performed with wandb and so you will need this set up on your machine in order to proceed. Note also that the output directory specified in the slurm template (see [requirements](#requirements) below.)
```

Comment:
revise wording of last sentence? 
"Note also <strike>that</strike> the output directory specified in the slurm template"?

Reply:
Oops, was supposed to finish off with " must exist prior to running the scripts."! Now done